### ISSUE TITLE
Resource Server configurers should pick up a JwtAuthenticationConverter bean

### ISSUE THREAD

**Issue by jzheaux** [spring-projects/spring-security#8185]
It's quite convenient to be able to expose `JwtDecoder` or `ReactiveJwtDecoder` as a bean and have Spring Security's OAuth 2.0 Resource Server pick it up.

It would be nice to be able to do the same with `JwtAuthenticationConverter`:

```java
@Bean 
JwtAuthenticationConverter jwtAuthenticationConverter() {
    JwtAuthenticationConverter converter = new JwtAuthenticationConverter();
    converter.setJwtGrantedAuthoritiesConverter(...);
    return converter;
}
```

**Comment by evgeniycheban** [comment]
I would like to take this task.

**Comment by akuma8** [comment]
@jzheaux Yes this could solve my issue where I have multiple filter chains but as a client I should not have to worry about that bean, I want to define it and forget it. I mean, I don’t want to inject that bean in each filter chain, spring security should handle it behind the scene.

**Comment by jzheaux** [comment]
Sure, @evgeniycheban! It's yours.

**Comment by akuma8** [comment]
Hi @jzheaux,
Is this feature released? If yes, which version of Spring Security? I would like to migrate my current configurations to this one. 
Also I would like to contribute again, if you have issues for new committers please let me kow.

**Comment by jzheaux** [comment]
Yes, @akuma8, it was released in 5.4.0-M1. The GA version (5.4.0) released in September.

It would be great to get more contributions from you! Please check for [the ideal-for-contribution label](https://github.com/spring-projects/spring-security/issues?q=is%3Aissue+is%3Aopen+label%3A%22status%3A+ideal-for-contribution%22) for some ideas. Currently, I don't see any OAuth 2.0 ones, but there's [a SAML 2.0 ticket](https://github.com/spring-projects/spring-security/issues/7722) there that would be helpful.
